File: HTML/META-INF/resources/dataforms/validator/DateValidator.js
/**
 * @fileOverview {@link DateValidator}クラスを記述したファイルです。
 */

/**
 * @class DateValidator
 * 日付バリデータクラス。
 * <pre>
 * </pre>
 * @extends DateTimeValidator
 */
DateValidator = createSubclass("DateValidator", {dateFormatKey: "format.datefield"}, "DateTimeValidator");

File: src/dataforms/validator/DateValidator.java
package dataforms.validator;

import dataforms.util.MessagesUtil;


/**
 * 日付バリデータクラス。
 *
 */
public class  DateValidator extends DateTimeValidator {
	/**
	 * コンストラクタ。
	 */
	public DateValidator() {
		super("error.date");
	}

	@Override
	public boolean validate(final Object value) {
		String format = MessagesUtil.getMessage(this.getPage(), "format.datefield");
		this.setDateFormat(format);
		return super.validate(value);
	}

}

